src/Tribe/Assets.php

```diff
@@ -75,6 +75,22 @@ public function enqueue_scripts() {
 						'tribe-tickets-rsvp',
 						'tribe-tickets-registration-page',
 					],
+				],
+			);
+
+			// @todo: Remove this once we solve the common breakpoints vs container based.
+			tribe_asset(
+				$tickets_main,
+				'tribe-common-responsive',
+				'common-responsive.css',
+				[ 'tribe-common-skeleton-style' ],
+				null,
+				[
+					'groups' => [
+						'tribe-tickets-block-assets',
+						'tribe-tickets-rsvp',
+						'tribe-tickets-registration-page',
+					]
```

Comment:
:no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 5b32de4c00ed235ddcdcf48c823355c9</sub>

Reply:
```suggestion
					],
```